web/func.go

```diff
@@ -108,6 +108,8 @@ func ParseTruthyFalsy(flag interface{}) (result bool, err error) {
 			result = true
 		case "false", "0":
```

Comment:
Could you have just added the `""` to this line instead of making a new `case`?

Also, mind adding a clause to `TestParseTruthyFalsy` for the changes?